test/schema/cockroachdb_specific_schema.rb

```diff
@@ -0,0 +1,122 @@
+# suite. Statements that don't work against CockroachDB have been commented out
+# with an explanation of why they don't work.
+
+ActiveRecord::Schema.define do
+
+  enable_extension!("uuid-ossp", ActiveRecord::Base.connection)
+  enable_extension!("pgcrypto",  ActiveRecord::Base.connection) if ActiveRecord::Base.connection.supports_pgcrypto_uuid?
+
+  uuid_default = connection.supports_pgcrypto_uuid? ? {} : { default: "uuid_generate_v4()" }
+
+  create_table :uuid_parents, id: :uuid, force: true, **uuid_default do |t|
+    t.string :name
+  end
+
+  create_table :uuid_children, id: :uuid, force: true, **uuid_default do |t|
+    t.string :name
+    t.uuid :uuid_parent_id
+  end
+
+  create_table :defaults, force: true do |t|
+    t.date :modified_date, default: -> { "CURRENT_DATE" }
+    t.date :modified_date_function, default: -> { "now()" }
+    t.date :fixed_date, default: "2004-01-01"
+    t.datetime :modified_time, default: -> { "CURRENT_TIMESTAMP" }
+    t.datetime :modified_time_function, default: -> { "now()" }
+    t.datetime :fixed_time, default: "2004-01-01 00:00:00.000000-00"
+    t.column :char1, "char(1)", default: "Y"
+    t.string :char2, limit: 50, default: "a varchar field"
+    t.text :char3, default: "a text field"
+    t.bigint :bigint_default, default: -> { "0::bigint" }
+    t.text :multiline_default, default: "--- []
+
+"
+  end
+
+  create_table :postgresql_times, force: true do |t|
+    t.interval :time_interval
+    t.interval :scaled_time_interval, precision: 6
+  end
+
+  create_table :postgresql_oids, force: true do |t|
+    t.oid :obj_id
+  end
+
+  drop_table "postgresql_timestamp_with_zones", if_exists: true
+  drop_table "postgresql_partitioned_table", if_exists: true
+  drop_table "postgresql_partitioned_table_parent", if_exists: true
+
+  execute "DROP SEQUENCE IF EXISTS companies_nonstd_seq CASCADE"
+  execute "CREATE SEQUENCE companies_nonstd_seq START 101 OWNED BY companies.id"
+  execute "ALTER TABLE companies ALTER COLUMN id SET DEFAULT nextval('companies_nonstd_seq')"
+  execute "DROP SEQUENCE IF EXISTS companies_id_seq"
+
+  # Stored procedures are not supported in CockroachDB.
+  # See https://github.com/cockroachdb/cockroach/issues/17511.
+  # execute "DROP FUNCTION IF EXISTS partitioned_insert_trigger()"
+
+  # CockroachDB uses unique_rowid() for primary keys by default instead of
+  # sequences. Therefore, there aren't any sequences to update here.
+  # See https://www.cockroachlabs.com/docs/v19.2/serial.html#modes-of-operation.
+  # %w(accounts_id_seq developers_id_seq projects_id_seq topics_id_seq customers_id_seq orders_id_seq).each do |seq_name|
+  #   execute "SELECT setval('#{seq_name}', 100)"
+  # end
+
+  execute <<_SQL
+  CREATE TABLE postgresql_timestamp_with_zones (
+    id SERIAL PRIMARY KEY,
+    time TIMESTAMP WITH TIME ZONE
+  );
+_SQL
+
+# Since stored procedures are not supported in CockroachDB, it won't be possible
```

Comment:
Should this be removed?

Reply:
Since this file is a copy of AR's PostgreSQL specific schema, I left stuff like this in place but commented out thinking it might make it easier to see the differences between the two files. We can take it out if it isn't helpful. What do you think?

Reply:
Yeah, this is ok to keep.